app/Console/Commands/RebuildImages.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Poniverse\Ponyfm\Console\Commands;
+
+use Illuminate\Console\Command;
+use Poniverse\Ponyfm\Models\Image;
+use Symfony\Component\HttpFoundation\File\File;
+
+class RebuildImages extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rebuild:images';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Resizes all images to fit the specifications in Models/Image';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $images = Image::all();
+        foreach ($images as $image) {
+            $this->info("Regenerating images for id:".$image->id. " (".$image->filename.")");
```

Comment:
It's customary to use progress bars instead of generating thousands of lines of output for CLI tasks that touch many records. Laravel provides a construct to make this easy: https://laravel.com/docs/5.3/artisan#writing-output

Only generate a line of output that refers to a specific image if there was an error processing it.